website/docs/learn/3-build-your-first-models.md

```diff
@@ -71,9 +71,9 @@ One of the most powerful features of dbt is that you can change the way a model
 ```yml
 models:
   jaffle_shop:
-    materialized: table
+    +materialized: table
```

Comment:
The one thing i'm not sure about is including it in the tutorial / learn onboarding — the [starter-project](https://github.com/fishtown-analytics/dbt-starter-project/blob/master/dbt_project.yml) generates a v1 syntax, and the videos are in v1 syntax. Not sure what to do here!

Reply:
Good point. Do we have a timeline on upgrading the starter project? I see that Jake added a [tag](https://github.com/fishtown-analytics/dbt-starter-project/releases/tag/dbt-yml-config-version-2) for config v2 projects, I'm guessing that's what `dbt init` on >=0.17.0 clones instead

Reply:
Just opened a [PR](https://github.com/fishtown-analytics/dbt-starter-project/pull/8) to fix the config on that repo 🙂 . I didn't realize we had updated the starter-script, I guess maybe it's a case of adding a little callout like "this differs from the video"?

Reply:
I ended up adding some info — once that renders correctly, let's merge this in :) 
(I have some other PRs I want to merge in tomorrow that are going to conflict big time!)